tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/decisiontree/ProbabilisticClassifier.scala

```diff
@@ -39,6 +38,11 @@ object ProbabilisticClassifier {
   implicit object ProbabilisticClassifierJsonFormat
       extends RootJsonFormat[ProbabilisticClassifier] {
 
+    // Override the Spray/JSON serialization for maps with integer keys, because these
+    // don't work out-of-the-box.
+    import spray.json.DefaultJsonProtocol.{ mapFormat => _, _ }
+    import org.allenai.nlpstack.parse.poly.decisiontree.DecisionTree.intMapFormat
+
     implicit val decisionTreeFormat =
       jsonFormat4(DecisionTree.apply).pack("type" -> "DecisionTree")
```

Comment:
This should be:
```
implicit val decisionTreeFormat = DecisionTree.dtFormat.pack("type" -> "DecisionTree")
```
This should also make it so you don't need the extra import above.